app/public/src/pages/component/game/game-pokemon-detail.css

```diff
@@ -19,8 +20,26 @@
   padding: 5px;
 }
 
-.game-pokemon-detail p {
-  margin: 0;
```

Comment:
this ended up being unnecessary because we have a rule for it already in `app\public\src\style\index.css`